### PR TITLE
view bus in route

### PR DESCRIPTION
**Pull request by NyiingangoPacifique**
### **`User view Buses in Choosen Route`**
pull this branch **git pull**

create .env file add **DB_URL** variable

**Add token in your .env file**

```
AUTH_SECRET = 
AUTH_EXPIRES = 1d
AUTH_ROUNDS = 10
ADMIN_SECRET_TOKEN = 
ADMIN_SECRET = 
OPERATOR_SECRET = 
DRIVER_SECRET =
```

install all required packages **npm install**
 run migrate **npm run migrate**
run seed **npm run seed**
run the server by **npm run dev**

Screenshot from 2022-03-13 13-47-59
Screenshot from 2022-03-13 13-48-22

### CODE REVIEW

**Review by JPaul23 — APPROVED** [review]

**Review by niyontwali — APPROVED** [review]
Thanks for the great work done.  You branch works and your endpoints are successfully executed. However, I can see you pulled your collegue's work. Make sure that your rebase once their work is approved. Well done!

**Review by christianR15c — APPROVED** [review]

**Review by christianR15c — APPROVED** [review]

File: test/role.js [hunk 1]
```diff
@@ -1,99 +1,98 @@
...
+//               response.should.have.status(200);
+//               response.body.should.be.a('object');
+//               response.body.should.have.property('message');
+//               if (error) console.log(error);
+//               done();
+//             });
+//         });
+//     });
+//   });
+// });
```
Comment (SimonImanigirimpuhwe) [inline test/role.js]: You need to remove commented code

File: test/role.js [hunk 2]
```diff
@@ -1,99 +1,98 @@
...
+//             .end((error, response) => {
+//               response.should.have.status(200);
+//               response.body.should.be.a('object');
+//               response.body.should.have.property('message');
+//               response.body.should.have.property('role');
+//               done();
+//             });
+//         });
+//     });
+//   });
```
Comment (SimonImanigirimpuhwe) [inline test/role.js]: This needs to be removed as well

**Review by SimonImanigirimpuhwe — CHANGES_REQUESTED** [review]

File: test/role.js (hunk 1 above)
Comment (NyiingangoPacifique) [reply test/role.js]: resolved now it clear.
thanks for your review

File: test/role.js (hunk 2 above)
Comment (NyiingangoPacifique) [reply test/role.js]: corrected

**Review by SimonImanigirimpuhwe — APPROVED** [review]

**Pull request merged**